pkg/security/probe/probe_windows.go

```diff
@@ -98,6 +104,16 @@ func (p *WindowsProbe) Start() error {
 					log.Errorf("unable to resolve parent pid %v", err)
 					continue
 				}
+				if start.RequiredSize != 0 {
+					// in this case, the command line and/or the image file might not be filled in
+					// depending upon how much space was needed.
+
+					// potential actions
+					// - just log/count the error and keep going
+					// - restart underlying procmon with larger buffer size, at least if error keeps occurring
```

Comment:
Should the buffer resize be marked as a TODO? I see we're logging the error and that is one of the potential actions you list, so I just want to clarify if the resize is coming in the future or if we can remove the comment entirely.